pyrefly/lib/test/pydantic/config.rs

```diff
@@ -36,6 +36,36 @@ m.x = 10 # E: Cannot set field `x`
 "#,
 );
 
+pydantic_testcase!(
+    test_model_config_dict_func,
+    r#"
+from pydantic import BaseModel
+
+class Model(BaseModel):
+    model_config = dict(frozen=True)
```

Comment:
Looks like this test is failing due to some missing TypedDict functionality in Pyrefly. I created https://github.com/facebook/pyrefly/issues/1248. Could you add an `# E:` comment here and link to that issue? I'll look into fixing it on Monday.